### DISCUSSION TITLE
在  简化直播间 去掉付费礼物时网页全面或者全屏会有一部分空白 建议去掉

### DISCUSSION THREAD

**Discussion by fuhahaha** [the1812/Bilibili-Evolved#4210] [功能建议 / Ideas]
网页全屏或者全屏后直播视频下面原本用来方付费礼物的地方还占着位置 会留一部分空白
![image](https://github.com/the1812/Bilibili-Evolved/assets/46596429/0cf92d15-6500-4fd5-8873-18aa5196155d)
把web-player__bottom-bar__container这个类去掉 把video标签里的height设置去掉去掉可以解决
![image](https://github.com/the1812/Bilibili-Evolved/assets/46596429/888bfd00-4c16-4372-92c4-7c0b624a9282)

**Comment by the1812** [comment]
可以先关闭简化直播间, 然后开全屏直播, 关闭礼物栏, 再重新打开简化直播间
